app/jobs/fixed_asset_export_job.rb

```diff
@@ -0,0 +1,22 @@
+class FixedAssetExportJob < ActiveJob::Base
+  queue_as :default
+  include Rails.application.routes.url_helpers
+
+  def perform(document_nature, key, state, period, user)
```

Comment:
Can we use like.. `nature` instead of `state`? or `kind`? The keyword `state` implies it's a particular.. well state, of the financial years involved.